server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java

```diff
@@ -38,4 +48,30 @@ public void afterAll() throws ExecutionException, IOException {
       server.shutdown();
     }
   }
+
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void createTableThenDropMetadataTableThenRepairTables_ShouldExecuteProperly()
+      throws Exception {
+    super.createTableThenDropMetadataTableThenRepairTables_ShouldExecuteProperly();
+  }
+
+  /** This test is disabled if {@link #isExternalServerUsed()} return true */
+  @Override
+  @Test
+  @DisabledIf("isExternalServerUsed")
+  public void
+      createTableThenDropMetadataTableThenRepairTablesWithCoordinator_ShouldExecuteProperly()
+          throws Exception {
+    super.createTableThenDropMetadataTableThenRepairTablesWithCoordinator_ShouldExecuteProperly();
+  }
```

Comment:
@Torch3333 Ditto. I think we also need to disable these methods if isExternalServerUsed is true, right?

Reply:
Yes, thank you for fixing this.